src/types.ts

```diff
@@ -1,13 +1,38 @@
-export type Post = {
+export type ApiPost = {
   id: string;
   title: string;
   slug: string;
   content: string;
-  bucket: string;
-  created: string;
   created_at: string;
-  created_by: string;
   modified_at: string;
   published_at: string;
-  type: 'posts';
+  metadata: {
+    soundcloud_id: number;
+    spanish_title: string;
+    spanish_slug: string;
+    spanish_content: string;
+    spanish_soundcloud_id: number;
+  };
+};
+
+export type Lang = 'en' | 'es';
+
+export type Post<L extends Lang> = {
+  id: string;
+  slug: string;
+  title: string;
+  content: string;
+  soundcloudId: number;
+  lang: L;
+  createdAt: Date;
+  modifiedAt: Date;
+  publishedAt: Date;
+};
+
+export type DualPost = {
+  en: Post<'en'>;
+  es: Post<'es'>;
+  createdAt: Date;
+  modifiedAt: Date;
+  publishedAt: Date;
```

Comment:
the timestamps are exposed on the root as well, for convenience, since they are always the same between both languages (they are edited as a unit inside cosmic)